File: testdata/jennies/rawtypes/field_with_struct_with_defaults/PHPRawTypes/src/Types/Defaults/DefaultsStructPartialComplexField.php
<?php

namespace Grafana\Foundation\Types\Defaults;

class DefaultsStructPartialComplexField implements \JsonSerializable {
    public string $uid;

    public int $intVal;

    /**
     * @return array<string, mixed>
     */
    public function jsonSerialize(): array
    {
        $data = [
            "uid" => $this->uid,
            "intVal" => $this->intVal,
        ];
        return $data;
    }
}

File: testdata/jennies/rawtypes/field_with_struct_with_defaults/CSharpRawTypes/src/Grafana/Foundation/Defaults/DefaultsStructPartialComplexField.cs
namespace Grafana.Foundation.Defaults;


public class DefaultsStructPartialComplexField
{
    public string Uid;
    public long IntVal;

    public DefaultsStructPartialComplexField()
    {
        this.Uid = "";
        this.IntVal = 0L;
    }

    public DefaultsStructPartialComplexField(string uid, long intVal)
    {
        this.Uid = uid;
        this.IntVal = intVal;
    }
}
